test/fw/ptl/utils/plugins/ptl_test_data.py

```diff
@@ -175,24 +178,26 @@ def __save_home(self, test, status, err=None):
             f.close()
             return
         pbs_snapshot_path = os.path.join(
-            server.pbs_conf["PBS_EXEC"], "sbin", "pbs_snapshot")
+            servers.values()[0].pbs_conf["PBS_EXEC"], "sbin", "pbs_snapshot")
         parent_dir = os.getcwd()
         cur_user = self.du.get_current_user()
         cmd = [
             pbs_snapshot_path,
+            '-H', server_host,
             '--daemon-logs',
             '2',
             '--accounting-logs',
             '2',
-            '--with-sudo']
+            '--with-sudo'
+            ]
         if len(add_hosts) > 0:
-            cmd += ['--additional-hosts=' + ','.join(map(str, add_hosts))]
-        cmd += ['-o', parent_dir]
-        cmd += ['> /dev/null', '2>&1']
-        ret = self.du.run_cmd(server_host, cmd, level=logging.DEBUG2)
+            cmd += ['--additional-hosts=' + ','.join(add_hosts)]
+        cmd += ['-o', pwd.getpwnam(cur_user).pw_dir]
+        ret = self.du.run_cmd(current_host, cmd, level=logging.DEBUG2,
```

Comment:
Any particular reason for setting level to logging.DEBUG2 ?

Reply:
Since, pbs_diag used to run with DEBUG2 logging, I have maintained the same.